_posts/2017-11-07-amp-roadshow.md

```diff
@@ -0,0 +1,114 @@
+[tweet-1]: https://twitter.com/wolframkriesing/status/927825238766882818
+[tweet-2]: https://twitter.com/wolframkriesing/status/927835789404319744
+[amp-validator]: https://validator.ampproject.org/
+
+## AMP in Production
+
+<div style="float: left; padding: 1rem;">
+<img src="{{site.baseurl}}/img/posts/2017-11-amp-roadshow/3-in-prod.jpg" alt="amp in production" width="100" class="sizeup-onhover-image scale3 origin-left-top" />
+<br/><em>AMP in production</em>
+</div>
+
+After the basics, the "why" and "how" of AMP, [Rudy][rudy] (???????? was it Rudy?????????) went to show us about how to get AMP live. Three basic steps he talked about 1) Validate 2) Distribute and 3) Measure. After an AMP page ran through the validator one wants to take it live. When this comes up the measuring part is an important one. Some sites need only simple page view tracking, others even want A/B tests. AMP tries to serve them all. There is a `<amp-analytics>` custom element that unifies the analytics work under one hood. It basically serves as a proxy on the client, I assume it has a fixed set of tracking data that it collects and does then spread it to the configured backends (analytics provider). This allows AMP to do the tracking and not every tracker to bring its own code, polute your site and potentially crash it.
+For the A/B testing `<amp-experiment>` can be used. As far as I have understood it's still limited and there might be complex scenarios that it doesn't serve yet.
+
+[rudy]: https://twitter.com/rudygalfi/
+
+## AMP and Ads
+
+As you can see in the [first commit][amp-first-commit] of the AMP project, ads was in there since the beginning. Which is of course not surprising for a company that makes most it's revenue with ads. In current websites ads are often one big slow down factor. It makes sense to tackle this problem. As with the analytics mentioned above AMP also provides strategies to deal with ads.
+
+<div style="float: left; padding: 1rem;">
+<img src="{{site.baseurl}}/img/posts/2017-11-amp-roadshow/4-ads-on-amp-pages.jpg" alt="Ads on AMP pages" width="100" class="sizeup-onhover-image scale3 origin-left-top" />
+<br/><em>Ads on AMP pages</em>
+</div>
+
+There are two ad strategies with AMP: 1) ads on AMP pages and 2) ads written in AMP. 
+The first one is the "simple" strategy, where you know the ad's size upfront and embed the ad using `<amp-ad>`. Doing this allows the page size to be determined upfront and won't make the site jump once the ad is loaded.
+
+<div style="float: right; padding: 1rem;">
+<img src="{{site.baseurl}}/img/posts/2017-11-amp-roadshow/5-ads-in-amp.jpg" alt="Ads written in AMP" width="100" class="sizeup-onhover-image scale3 origin-right-top" />
+<br/><em>Ads written in AMP</em>
+</div>
+
+The other ad strategy is to build ads using AMP. Which means building your ads yourself by just using AMP elements.
+Which then will ensure the speed that AMP tries to ensure. The embeddability is another factor, which is another of the 
```

Comment:
Confusing: `Which then will ensure the speed that AMP tries to ensure.`, not sure what you want to say here 🙂